shell/common/shell.cc

```diff
@@ -1240,4 +1251,17 @@ Rasterizer::Screenshot Shell::Screenshot(
   return screenshot;
 }
 
+bool Shell::WaitForFirstFrame(fml::TimeDelta timeout) {
```

Comment:
I believe this will deadlock on iOS when there is a platform view composited inline. In that configuration, the GPU task runners and the platform task runners are the same. The DCHECKs will also trip in that case.

Reply:
Done.